stories/typography.js

```diff
@@ -21,19 +22,19 @@ storiesOf('Typography', module)
   .addDecorator(checkA11y)
   .addDecorator(styles(baseStyles))
   .add('Headings', () => (
-    <div>
-      <Heading1>Heading 1</Heading1>
-      <Heading2>Heading 2</Heading2>
-      <Heading3>Heading 3</Heading3>
-      <Heading4>Heading 4</Heading4>
-    </div>
+    <Box>
+      <Heading1 marginTop={4}>Heading 1</Heading1>
+      <Heading2 marginTop={4}>Heading 2</Heading2>
+      <Heading3 marginTop={4}>Heading 3</Heading3>
+      <Heading4 marginTop={4}>Heading 4</Heading4>
+    </Box>
   ))
   .add('Text', () => (
-    <div>
-      <Monospaced>Monospaced</Monospaced>
-      <OldStyleNumber>Old style number</OldStyleNumber>
+    <Box>
       <TextBody>Text body</TextBody>
-      <TextSmall>Text small</TextSmall>
-      <TextTiny>Text tiny</TextTiny>
-    </div>
+      <TextSmall marginTop={2}>Text small</TextSmall>
+      <TextTiny marginTop={2}>Text tiny</TextTiny>
+      <TextBody marginTop={4}><Monospaced>Monospaced number 1234567890</Monospaced></TextBody>
+      <TextBody marginTop={2}><OldStyleNumber>Old style number 1234567890</OldStyleNumber></TextBody>
+    </Box>
```

Comment:
Maybe add another story to show the overriding of the general margin/padding. Something like `margin={4} margin-top={0}`.

Reply:
I fixed @nickwaelkens first remark with the default values. Now I can something like `margin={4} margin-top={0}`and it works!

 :ok: by @driesd on 19/10/2017 15:53:52

Reply:
I implemented @nickwaelkens his solution (first remark) with the default values. Now you can perfectly override any margin/padding with a value of 0. So, right now, padding={4} paddingTop={0} works very well.

 :ok: by @driesd on 19/10/2017 16:18:48

Reply:
Yes, looks good. Tested it again and works perfectly now 💅 

 :ok: by @lowiebenoot on 19/10/2017 16:51:19